src/lib/jobBoard.ts

```diff
@@ -11,15 +11,15 @@ interface JobBoardSpec {
 
 export const JOB_BOARD_SPEC: { [jobBoard in JobBoard]: JobBoardSpec } = {
     [JobBoard.coop]: {
-        name: 'Co-op',
-        url: 'https://waterlooworks.uwaterloo.ca/myAccount/co-op/coop-postings.htm',
+        name: 'Co-op - Full-cycle Service',
```

Comment:
Can we leave this as "Co-op"? The text is used here: 
<img width="309" alt="image" src="https://github.com/uwwave/extension/assets/16693230/29a69cb8-b4ca-43db-8d8a-6ff5cc722ff9">

Reply:
I see your point... I think making it "Co-op Cycle" would be more clear for users while still saving spaces. I'll update this.